library/Recaptcha.php

```diff
@@ -182,19 +182,19 @@ public function create_box($attr = NULL)
         }
 
         // Check if there are attributes passed
-        if($attr === NULL){
+        if($attrib === NULL){
             // No attributes were passed add the g-recaptcha class
             $box .= ' class="g-recaptcha"';
         }else{
             // Attributes are passed, check if there is a class attribute
-            if( empty($attr['class']) ){
+            if( empty($attrib['class']) ){
                 // No class attribute was passed
                 // Add the g-recaptcha class
-                $attr['class'] = 'g-recaptcha';
+                $attrib['class'] = 'g-recaptcha';
             }else{
                 // There is a class attribute passed
                 // Add g-recaptcha to the previous value
-                $attrib['class'] .= 'g-recaptcha';
+                $attrib['class'] .= ' g-recaptcha';
```

Comment:
Instead of changing all `$attr` to `$attrib`, doing the opposite is better.

Reply:
When use multi class, $attrib[‘class’] will concat string like “class1class2”. So I add space between them.

Reply:
You use $attr in all place except line 197 and in foreach loop you use $attrib as $attr. It not correct.

Reply:
I see the problem, it can be good to go once you do the following edits:
- Instead of changing `$attr` to `$attrib`, change `$attrib` in line 197 and 201, in line 201 change `$attr` to `$attrib`
- Revert the change you did in the `is_valid` method

Reply:
Ok, deal.